2_files.py

```diff
@@ -11,12 +11,32 @@
 5. Сохраните результат в файл referat2.txt
 """
 
+
 def main():
-    """
-    Эта функция вызывается автоматически при запуске скрипта в консоли
-    В ней надо заменить pass на ваш код
-    """
-    pass
+    with open('referat.txt', 'r', encoding='utf-8') as f:
+        text = f.read()
+        print(text)
+        line_length = len(text)
+        print(line_length)
+        word_count = text.count(' ')
```

Comment:
как вариант, но двойные пробелы он будет считать за два слова, и даже если все пробелы одинарные, результат будет на 1 слово меньше, чем должно быть
```suggestion
        words = text.split()
        word_count = len(words)
```